src/plugins/vis_type_vislib/public/vislib/lib/axis/axis_scale.js

```diff
@@ -49,10 +49,19 @@ export class AxisScale {
     });
   }
 
-  getTimeDomain(data) {
+  getDomainExtent(data) {
     return [this.minExtent(data), this.maxExtent(data)];
   }
 
+  getOrdinalDomain(values = []) {
+    if (this.ordered?.interval !== undefined) {
+      const [min, max] = this.getDomainExtent(values);
+      return _.range(min, max + this.ordered.interval, this.ordered.interval);
```

Comment:
For example...

```ts
// if your values looked like this
const values = [10, 30, 40, 60];

// this would just fill in the gaps based on the min, max and interval
const values = [10, 20, 30, 40, 50, 60];
```